protocol/communities/manager.go

```diff
@@ -4189,6 +4189,7 @@ func (m *Manager) GetOwnedCommunitiesChatIDs() (map[string]bool, error) {
 			for _, id := range c.ChatIDs() {
 				chatIDs[id] = true
 			}
+			chatIDs[c.MemberUpdateChannelID()] = true // TODO: for now including this chatID as controlled so that archiving works without any issues.
```

Comment:
That's a bit unexpeced as for the function that is supposed to get channels (ChatIDs). Would it be possible to move it to place where it is indeed needed?

Reply:
good point, let me see if i can move this change to the source of where this is getting used.

Reply:
ah, realized that https://github.com/status-im/status-go/blob/d61d20042aa8956ce8634da0ee5f603df379d6dc/protocol/messenger.go#L3589 is the place where i don't have access to list of communities in order to get this specific channel. 

One way i can think of approaching this is to add another method to fetch list of univeral-channels for all owned communities and then invoke it at the source.

Reply:
Segregated this to a separate method.